R/loo.R

```diff
@@ -59,7 +59,13 @@
 #' @param k_threshold Threshold for flagging estimates of the Pareto shape
 #'   parameters \eqn{k} estimated by \code{loo}. See the \emph{How to proceed
 #'   when \code{loo} gives warnings} section, below, for details.
-#'
+#' @param r_eff \code{TRUE} or \code{FALSE} indicating whether to compute the
+#'   \code{r_eff} argument to pass to the \pkg{loo} package. If \code{TRUE},
+#'   \pkg{rstanarm} will call \code{\link[loo]{relative_eff}} to compute the
+#'   \code{r_eff} argument to pass to the \pkg{loo} package. If \code{FALSE}
+#'   (the default), we avoid computing \code{r_eff}, which can be very slow. The
```

Comment:
Maybe add here before the last sentence something like
`r_eff` measures the amount of autocorrelation in MCMC draws, and is used to compute more accurate ESS and MCSE estimates for pointwise and total elpds.